.github/workflows/check.yaml

```diff
@@ -14,3 +14,13 @@ jobs:
           cache: npm
       - run: npm ci
       - run: npm run lint
+  link-check:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: npm
+      - run: npm ci
+      - run: npm run markdown-link-check
```

Comment:
suggestion (non-blocking): I think proliferating a pattern where some jobs are run in github actions and some are run in prow will lead to everything being harder to maintain in the future.

I'd rather not use github actions at all, personally, but more important than "actions vs prow" is consistency.

I don't think it's a blocker, because I don't want to get in the way of a good PR... but I think it's a concern.